sdk/scheduler/src/main/java/com/mesosphere/sdk/dcos/Capabilities.java

```diff
@@ -32,9 +32,9 @@ public boolean supportsGpuResource() throws IOException {
         return hasOrExceedsVersion(1, 9);
     }
 
-    public boolean supportCniPortMapping() throws IOException {
+    public boolean supportsCNINetworking() throws IOException {
         // CNI port-mapping is supported by DC/OS 1.10 upwards
-        return hasOrExceedsVersion(1, 10);
+        return hasOrExceedsVersion(1, 9);
```

Comment:
Could add an override to `SchedulerFlags`? E.g. an envvar like `FORCE_CNI_SUPPORT=[true|false]` that when provided forces support to be enabled or disabled regardless of the detected DC/OS version